/**:
  ros__parameters:
    map_pcd_file: "/home/rubis/AutowareAuto/install/rubis_0/share/rubis_0/data/autonomoustuff_parking_lot_lgsvl.pcd"
    map_yaml_file: "/home/rubis/AutowareAuto/install/rubis_0/share/rubis_0/data/autonomoustuff_parking_lot_lgsvl.yaml"
    map_frame: "map"
    map_config:
      capacity: 1000000
      min_point:
        x: -1000.0
        y: -1000.0
        z: -3.0
      max_point:
        x: 1000.0
        y: 1000.0
        z: 3.0
      voxel_size:
        x: 3.5
        y: 3.5
        z: 3.5
    viz_map: True
